generators/client/templates/vue/src/main/webapp/app/main.ts.ejs

```diff
@@ -130,3 +132,29 @@ new Vue({
 <%_ } _%>
   store,
 });
+
+setupAxiosInterceptors(
```

Comment:
Why not in the config package?

Reply:
Implemented here because it needs vue, store and loginService instances.
But it's possible to pass by parameters.